standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/jdbc/functions/AbortCompactionFunction.java

```diff
@@ -152,7 +150,7 @@ public CompactionAborter withCompactionInfo(CompactionInfo compactionInfo) {
 
     @Override
     public AbortCompactionResponseElement execute() {
-      try (TransactionContext context = jdbcResource.getTransactionManager().getNewTransaction(PROPAGATION_REQUIRED)) {
+      try (TransactionContext context = jdbcResource.getTransactionManager().getNewTransaction(null)) {
```

Comment:
passing `null` is not a good practice

Reply:
I tried to create an instance of `Transactional`, but it's hard to implement an annotation